.gitmodules

```diff
@@ -29,3 +29,6 @@
 [submodule "packages/contracts-bedrock/lib/openzeppelin-contracts-v5"]
 	path = packages/contracts-bedrock/lib/openzeppelin-contracts-v5
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "packages/contracts-bedrock/lib/solady-v0.0.245"]
+	path = packages/contracts-bedrock/lib/solady-v0.0.245
+	url = https://github.com/vectorized/solady
```

Comment:
Why did we decide to add a second version rather than update the other one?

Reply:
The update implies touching other contracts out of scope, here is the thread where we took the decision https://discord.com/channels/1244729134312198194/1290682033588404298